java/gandiva/src/main/java/org/apache/arrow/gandiva/evaluator/JniLoader.java

```diff
@@ -69,15 +70,30 @@ private static JniLoader setupInstance() throws GandivaException {
 
   private static void loadGandivaLibraryFromJar(final String tmpDir)
           throws IOException, GandivaException {
-    final String libraryToLoad = System.mapLibraryName(LIBRARY_NAME);
-    final File libraryFile = moveFileFromJarToTemp(tmpDir, libraryToLoad);
+    final String libraryToLoad =
+        getNormalizedArch() + File.separator + System.mapLibraryName(LIBRARY_NAME);
+    final File libraryFile = moveFileFromJarToTemp(tmpDir, libraryToLoad, LIBRARY_NAME);
     System.load(libraryFile.getAbsolutePath());
   }
 
+  private static String getNormalizedArch() {
+    String arch = System.getProperty("os.arch").toLowerCase(Locale.US);
+    switch (arch) {
+    case "amd64":
+      arch = "x86_64";
+      break;
+    case "aarch64":
+      arch = "aarch_64";
+      break;
+    default:
+      break;
+    }
+    return arch;
+  }
```

Comment:
```suggestion
  private static String getNormalizedArch() {
    String arch = System.getProperty("os.arch").toLowerCase(Locale.US);
    switch (arch) {
      case "amd64":
        arch = "x86_64";
        break;
      case "aarch64":
        arch = "aarch_64";
        break;
      default:
        break;
    }
    return arch;
  }
```

Reply:
Sorry! Could you push these style fixes to this branch and re-run the `java-jars` job?

Reply:
Sure!